### ISSUE TITLE
It's too hard to find Windows 64bit installers

### ISSUE THREAD

**Issue by sscherfke** [python/pythondotorg#358]
By default, pyhton.org will always pre-select the 32-bit installer for the current Python release, even for users of a 64bit Windows.

It requires at least 3 clicks to find a page that displays actual download links for other installers - if you know where you want to go. If you are new, you'll have a very hard time finding the 64bit-Installer for the latest release. This was definately easier on the old website.

There is also now generic URL that you could point to in a tutorial (like python.org/download/3.4 which points to the latest release of 3.4). The "best" URL that I found is https://www.python.org/downloads/release/python-340/, but I do not want to use it because I'd have to update it on every minor release.

To sum up:
1. Make it easiers for users get a list of all installers for a certain Python release.
2. Provide generic links to the latest releases (e.g., python.org/downloads/3/ and python.org/downloads/2)

**Comment by frankwiles** [comment]
We have /downloads/latest/python2/ and /downloads/latest/python3/ now to make this easier.
